app/editor/components/Editor.tsx

```diff
@@ -56,12 +67,7 @@ export default function PrimeReactEditor() {
       <Editor
         value={postBody}
         onTextChange={(e: any) => setPostBody(e.htmlValue)}
-        style={{
-          height: "50vh",
-          display: postBody && "flex",
-          justifyContent: postBody && "center",
-          cursor: "text",
-        }}
+        style={editorStyle}
```

Comment:
How about putting this style into css file and using a classname?

Reply:
I'll fix it